datafusion/physical-plan/src/aggregates/group_values/column_wise.rs

```diff
@@ -154,7 +187,9 @@ impl GroupValues for GroupValuesColumn {
                         let b = ByteGroupValueBuilder::<i64>::new(OutputType::Binary);
                         v.push(Box::new(b) as _)
                     }
-                    dt => todo!("{dt} not impl"),
+                    dt => {
+                        return internal_err!("{dt} not supported in GroupValuesColumn")
```

Comment:
```suggestion
                        return not_impl_err!("{dt} not supported in GroupValuesColumn")
```
It was todo!() previously, which I assume means we might implement it in the future. How about using `not_impl_err` instead?